Bottom: 6a6183b76ec5f92f7e57d9e41cd4f4b0b2a0b119
Top:    8a366620bce2a94c5302aff64691bc3c990516c0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 01:38:37 +0000

Refresh of find-c-pass-flags-to-sub-funct

---

diff --git a/src/find.c b/src/find.c
index c1151fa..dcf9761 100644
--- a/src/find.c
+++ b/src/find.c
@@ -281,7 +281,6 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-static int TheFlag = FOUNDFLAG;
 static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
 static bool User = false;    /* user action causing this */
